frontend/src/app/main/ui/ds/foundations/typography.mdx

```diff
@@ -0,0 +1,135 @@
+import { Canvas, Meta } from "@storybook/blocks";
+import * as TypographyStories from "./typography.stories";
+
+<Meta of={TypographyStories} />
+
+## Fonts
+
+Our preferred font is "Work Sans", it is used throughout our interface.  
+In case of having the interface in Farsi or Arabic this font will be replaced by "Vazirmatn".
+
+In the code area "Roboto Mono" will be used.
+
+## Variants
```

Comment:
Let's also add a technical note section speaking of the typography IDs devs can import, so no need to type the string.